src/components/Trade/TradeInput.tsx

```diff
@@ -78,6 +81,19 @@ export const TradeInput = ({ history }: RouterProps) => {
     if (!wallet) return
     if (!(quote?.sellAsset && quote?.buyAsset && quote.sellAmount)) return
 
+    const minSellAmount = toBaseUnit(quote.minimum, quote.sellAsset.precision)
+
+    if (bnOrZero(quote.sellAmount).lt(minSellAmount)) {
+      toast({
```

Comment:
in a follow-up PR we could look at making `showErrorToast` support passing in translation strings in addition to errors so we can replace this `toast` call with that.